app/src/main/res/layout/activity_login.xml

```diff
@@ -3,6 +3,7 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:fitsSystemWindows="true"
+    android:id="@+id/nsv_bkg"
```

Comment:
what does the `nsv_bkg` stand for? Could you please give it's full name?

Reply:
It's NestedScrollView which is the background. I found places where if a LinearLayout is used as a login tray its id is given as "ll_login" and hence I gave it an id "nsv_bkg", should I rename it?

Reply:
Yes, something to: `nsv_background`

Reply:
done!!